docs/advanced/UsageWithReactRouter.md

```diff
@@ -204,6 +217,22 @@ const App = ({ params }) => {
 };
 ```
 
+#### `index.js`
+```js
+import React from 'react';
+import { render } from 'react-dom'
+import { createStore } from 'redux'
+import todoApp from './reducers'
+import Root from './components/Root'
+
+let store = createStore(todoApp)
+
+render(
+	<Root store={store} />,
+	document.getElementById('root')
+)
+```
+
```

Comment:
This belongs up below the `components/Root.js` definition. It doesn't have any relation to this section of the doc.